src/components/dialogs/substation-creation-dialog.js

```diff
@@ -58,13 +60,18 @@ const NonNullStringPair = ({
         return Object.keys(predefined ?? {}).sort();
     }, [predefined]);
 
+    const entryToValue = (entry) => {
+        return func_identity(entry);
+    };
     const [name, nameField] = useAutocompleteField({
         id: 'pairKey' + index,
         label: 'PropertyName',
         formProps: filledTextField,
         validation: { isFieldRequired: true },
         values: predefinedNames,
         allowNewValue: true,
+        getLabel: getIdOrSelf,
+        newEntryToValue: entryToValue,
```

Comment:
Just give func_identity for newEntryToValue

Reply:
Fixed